README.md

```diff
@@ -235,15 +241,15 @@ Please note the static imports of the [assertions entry point](https://joel-cost
 and the order of `...Assertions.assertThat;`.
 
 ```java
-import static com.foursoft.kblmodel.kbl24.assertions.Assertions.assertThat;
+import static com.foursoft.harness.kbl.kbl-v24.assertions.Assertions.assertThat;

```

Comment:
Wouldn't that be `import static com.foursoft.harness.kbl.v24.assertions.Assertions.assertThat;`? Without the `kbl-`?